pex/resolve/resolver_options.py

```diff
@@ -119,8 +120,9 @@ def register(
             default=None,
             type=str,
             help=(
-                "Resolve requirements from the given lock file instead of from --index servers, "
-                "--find-links repos or a --pex-repository."
+                "Resolve requirements from the given PEX lock file instead of from --index "
```

Comment:
PEX file is right above, a.pex is a PEX file, but Pex lock file is right here, this is a lock file created by Pex (the tool).